### DISCUSSION TITLE
Show audio output type next to name in output selector

### DISCUSSION THREAD

**Discussion by sevmonster** [jcorporation/myMPD#999] [Ideas]
This might not be a common situation, but for me I have a lot of outputs in MPD: `shout`, `pipewire`, `pulse`, `pipe`...

It would be nice to have a bit more differentiation between outputs in the selector, by showing the `type` in addition to the name. This could be text prepended to the name, a separately styled chip/tag, or maybe an icon next to each output.

I don't know if the type is even exposed by the `mpd` API. If it's not, nevermind.

**Comment by jcorporation** [comment]
The response contains the output plugin. myMPD uses this already to differentiate between output types, this can be improved.
